l10n/en-US.ftl

```diff
@@ -12,7 +12,7 @@ baseline-supported-in = Supported in { $browsers }
 baseline-unsupported-in = Not widely supported in { $browsers }
 baseline-supported-and-unsupported-in = Supported in { $supported }, but not widely supported in { $unsupported }
 
-homepage-hero-title = Resources for <u data-l10n-name="developers">Developers</u>,<br> by Developers
+homepage-hero-title = Resources for Developers,<br /> by Developers
```

Comment:
Avoid closing void tags:

```suggestion
homepage-hero-title = Resources for Developers,<br> by Developers
```

Reply:
@caugner, isn’t it what we do with Prettier by default for all MDN content/code?

See examples https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/br#try_it